composer.json

```diff
@@ -117,6 +117,7 @@
         "league/flysystem-google-cloud-storage": "^3.0",
         "league/iso3166": "^4.2.1",
         "litipk/php-bignumbers": "^0.8.6",
+        "mgdsoft/symfony-profiler-spa": "^1.0",
```

Comment:
I think this should be as a dev dependency and also added in the project-base's composer.json file, right?